### PR TITLE
Update Paint version to 0.6.1

### PR DESCRIPTION
**Pull request by katherinlaine**
## What's Up
New Paint version available, so updating paint-rails with Paint 0.6.1.

Also adding instructions for updating to the Readme!

### CODE REVIEW

**Comment by katherinlaine** [comment]
@daemonsy - think i got it working! And added the Readme. Let me know if you have any suggestions.

File: README.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ Or install it yourself as:
 
 TODO: Write usage instructions here
 
+### Update Paint Version
+1. Clone paint-rails
+2. Update lib/paint-rails/version.rb with updated version number.
+3. git submodule init
+4. git submodule update
+5. Update bower.json file to reflect updated version (in /vendor/paint)
```
Comment (daemonsy) [inline README.md]: Should we be updating Paint::Rails::VERSION to reflect the bower.json version instead?

**Comment by daemonsy** [comment]
Other than the comment. :+1: 

Will merge and build new version when it's up. By the way, have you tested this gem locally on reporting? (Do remember this is a public conversation :smile:)

**Comment by katherinlaine** [comment]
hm, have not tested the gem locally with reporting yet. shall i do that before you build?

File: README.md [hunk 2]
```diff
@@ -22,6 +22,16 @@ Or install it yourself as:
 
 TODO: Write usage instructions here
 
+### Update Paint Version
+1. Clone paint-rails
+2. git submodule init
+3. git submodule update
+4. Update vendor/paint/bower.json to reflect updated version.
```
Comment (daemonsy) [inline README.md]: I think step 7 is valid, but we don't need to update the bower.json on the pain repo. It's our vendor lib and we should not change anything on it.

**Comment by katherinlaine** [comment]
good to go!

File: README.md [hunk 3]
```diff
@@ -22,6 +22,15 @@ Or install it yourself as:
 
 TODO: Write usage instructions here
 
+### Update Paint Version
+1. Clone paint-rails
+2. git submodule init
+3. git submodule update
+4. In paint directory, checkout master branch and 'git pull' changes.
+6. Update lib/paint-rails/version.rb with updated version number.
```
Comment (daemonsy) [inline README.md]: The ordering - 4,6,5 :smile:

**Comment by deioo** [comment]
0.6.2 is out already :)

**Pull request merged**